tests/test_other.py

```diff
@@ -6387,12 +6418,12 @@ def test_function_eliminator_double_parsed_correctly(self):
       tools.duplicate_function_eliminator.run(output_file)
 
       # Run last opts
-      tools.shared.safe_copy(tools.js_optimizer.run(output_file, ['last', 'asm']), output_file)
-      output_file_contents = self.get_file_contents(output_file)
+#      tools.shared.safe_copy(tools.js_optimizer.run(output_file, ['last', 'asm']), output_file)
```

Comment:
Err, amazing catch(!) :+1:  These were accidentally left over from debugging which call was doing the leaking, and lost sight of this swallowed by the change in indentation. Fixed